keyboards/45rstandard/keymaps/suliveevil/keymap.c

```diff
@@ -0,0 +1,150 @@
+#include "kb.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC, 
+		KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_ENT, 
+		KC_LSFT, LT(4, KC_NO), KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_SLSH, KC_UP, KC_RCTL, 
+		KC_LCTL, KC_LGUI, KC_LALT, LT(0, KC_NO), KC_SPC, LT(1, KC_NO), LT(2, KC_NO), KC_LEFT, KC_DOWN, KC_RGHT),
+
+	KEYMAP(
+		KC_TRNS, KC_1, KC_2, KC_3, KC_4, KC_LBRC, KC_RBRC, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_5, KC_6, KC_7, KC_8, KC_SCLN, KC_QUOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_9, KC_0, KC_MINS, KC_EQL, KC_COMM, KC_DOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		RESET, KC_TRNS, KC_TRNS, LT(3, KC_NO), KC_RSFT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	KEYMAP(
```

Comment:
```suggestion
	LAYOUT(
```

Reply:
You may also want to change all the `KC_TRNS` keys in your keymap to `_______` as it makes it much clearer.